profiles/common/modules/custom/tmgmt_poetry/tmgmt_poetry_mock/includes/tmgmt_poetry_mock_dashboard.inc

```diff
@@ -144,14 +144,20 @@ function _poetry_mock_get_data_row_action_links($data_row) {
       . $data_row['target_lg'] . '/'
       . $data_row['file_id'] . '/'
       . $data_row['tjiid'],
-      'query' => array('destination' => current_path()),
+      'query' => ['destination' => current_path()],
     ],
     'refuse' => [
       'title' => 'Refuse',
       'href' => 'tmgmt_poetry_mock/refuse/'
       . $data_row['file_id'] . '/'
       . $data_row['tjiid'],
-      'query' => array('destination' => current_path()),
+      'query' => ['destination' => current_path()],
+    ],
+    'send_ong_status' => [
+      'title' => "Send 'ONG' status",
+      'href' => 'tmgmt_poetry_mock/send-status/ong/'
+      . $data_row['tjid'],
```

Comment:
But I see it has been done the same way elsewhere in the code, so let's keep it to be consistent for now.